analyzers/Papas.py

```diff
@@ -70,8 +70,9 @@ def process(self, event):
             self.display.clear()
         pfsim_particles = []
         gen_particles = getattr(event, self.cfg_ana.gen_particles)
+        enable_matter_scattering = self.cfg_ana.enable_matter_scattering
         try: 
-            self.simulator.simulate( gen_particles )
+            self.simulator.simulate( gen_particles, enable_matter_scattering )
```

Comment:
if line is too long after accounting for previous comment, you may add a line break after the comma